var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -210,6 +327,7 @@ def build_nmodl(self, spec, prefix):
         make()
```

Comment:
I think this is dangerous w.r.t. build failures. Is there any way to separate `make` and `make install`?

Reply:
You are right. I think it's possible to refactor a bit the `autotools` build and add all the `make()` calls at least in the `build` stage.